src/StoreApi/docs/rate-limiting.md

```diff
@@ -0,0 +1,90 @@
+# Rate Limiting for Store API endpoints  <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+
+- [Limit information](#limit-information)
+- [Methods restricted by Rate Limiting](#methods-restricted-by-rate-limiting)
+- [Rate Limiting options filter](#rate-limiting-options-filter)
+- [Proxy standard support](#proxy-standard-support)
+- [Limit usage information observability](#limit-usage-information-observability)
+    - [Response headers example](#response-headers-example)
+- [Tracking limit abuses](#tracking-limit-abuses)
+    - [Custom tracking usage example](#custom-tracking-usage-example)
+
+[Rate Limiting](https://github.com/woocommerce/woocommerce-blocks/pull/5962) can now be enabled for Store API endpoints.
```

Comment:
We should probably rewrite this in present tense, "can now be enabled for Store API endpoints." feels more like an announcement and not a doc page.

Reply:
We should mention that it's opt in by default.